tests/test_deserialization.py

```diff
@@ -1428,7 +1428,7 @@ class MySchema(Schema):
             MySchema().load({"email": "foo"})
         errors = excinfo.value.messages
         assert len(errors["email"]) == 2
-        assert "Not a valid email address." in errors["email"][0]
+        assert any("Not a valid email address." in e for e in errors["email"])
```

Comment:
I made the tests not care about error message ordering. Is the order relevant? If yes, I guess we can revert the test changes and change the `self.validators.append(validator)` lines to `self.validators.insert(0, validator)`

Reply:
I believe it was intentional for the `Email` and `URL` validators to run first, which is why the code did `.insert` rather than `.append`. Can you revert the change?

Reply:
:+1: reverted